app/models/user_detail.rb

```diff
@@ -0,0 +1,16 @@
+class UserDetail
```

Comment:
love this. i think maybe `GithubUser` might be a better name? not sure though, just an idea 👍